inception-project/inception
# #1009 Improve style of document details page

**What's in the PR**
* resolves #1009 

**How to test manually**
* execute external search
* view document details page
* notice that there is a margin and newlines are displayed correctly

**Automatic testing**

**Documentation**
